src/content/docs/tutorial-infrastructure-monitoring/explore-host-data.mdx

```diff
@@ -0,0 +1,124 @@
+---
+
+import hostSummaryPage from 'images/infrastructure_screenshot-full_hosts-summary-page.webp'
+
+import metrics1 from 'images/infrastructure_screenshot-full_metrics-time1.webp'
+
+import metrics2 from 'images/infrastructure_screenshot-full_metrics-time2.webp'
+
+import summaryTableAlerting from 'images/infrastructure_screenshot-full_summary-table-alerting.webp'
+
+The performance of your infrastructure determines how well your APIs, services, and apps can make and respond to requests. A hardware failure can prevent your server from handling data, or maybe a memory leak brings an otherwise performant app to an abrupt halt. 
+
+Like the bridges and roads of a dense city, your system infrastructure needs regular inspection and maintenance. What processes affect CPU and disk usage? What app behaviors typically correlate with failing hosts? When you're able to observe your infrastructure in context, you can respond to incidents with more clarity and direction. Eventually, you'll be able to act before the incident even occurs.  
+
+## Objectives [#objectives]
+
+This tutorial series focuses on monitoring your system infrastructure. By the end of the tutorial series, you will learn to:
+
+* Analyze the data reported by our infrastructure agent
+* Act on patterns you find in our UI
+* Share your findings with other teams, like dev or sysadmin teams
+
+## Explore your host data [#explore]
+
+<Steps>
+    <Step>
+    ### Install the infrastructure agent 
+
+To install the infrastructure agent, we recommend using our [guided install](https://one.newrelic.com/nr1-core?state=a36f15a1-c5ce-528c-2ec2-8bcaacc40c03). Our guided install is a CLI installer that allows New Relic to inspect, then make recommendations about additional instrumentation to your system. For example, let's say you want to instrument your Apache hosts, which support these apps and services:
+
+* A few Springboot apps running in JVMs
+* Some microservices written in Ruby
+* Some microservices written in Go 
+
+In this case, the guided install would recommend:
+
+* Our infrastructure agent for your Apache VMs
+* The Java APM agent for your Springboot app
+* Additional APM installs for your Ruby and Go microservices
+
+You can choose to install the infrastructure agent only, or some combination of these recommendations. To get the most out of New Relic and this tutorial, we recommend you instrument anything you want to monitor.
+
+<Callout variant="important">
+While New Relic offers a full suite of tools and integrations to monitor Cloud infrastructure, Kubernetes, and other infra software, this tutorial is specific to data reported out of the box by our infra agent.
+</Callout>
+        
+    </Step>
+    <Step>
+
+Once installed, you can find your infrastructure data in our UI. Go to **[one.newrelic.com > All Capabilities > Infrastructure > Hosts](https://one.newrelic.com/infra/infrastructure/hosts)**. You're now looking at a summary page about your instrumented hosts. 
```

Comment:
you don't have to click `Hosts` anymore here

`one.newrelic.com > All Capabilities > Infrastructure`